tileview/src/main/java/com/qozix/tileview/graphics/BitmapRecyclerDefault.java

```diff
@@ -0,0 +1,10 @@
+package com.qozix.tileview.graphics;
+
+import android.graphics.Bitmap;
+
+public class BitmapRecyclerDefault implements BitmapRecycler {
+  @Override
+  public void recycleBitmap(Bitmap b ) {
+    b.recycle();
```

Comment:
i'd probably move the `isRecycled` check here, in case a particular implementation didn't care, or wanted to act on the opposite state...
```
if( !bitmap.isRecycled() ) {
  bitmap.recycle();
}
```